### ISSUE TITLE
Sync Notion database with Strapi CMS

### ISSUE THREAD

**Issue by gupta-ji6** [scriptified/scriptified.dev#85]
We currently store all our resources in Notion and use Strapi as CMS. We create draft issue in Notion and after review & approval of both @prateek3255 & @gupta-ji6, we add them on Strapi CMS again.

If we sync Notion with Strapi using Notion API and Strapi API/Webhooks, our duplicated work will be automated 🛠️

**Comment by hussamkhatib** [comment]
Doesn't strapi itself have that feature where a document is initially saved as a draft and you  publish it whenever you want.
Both of you can review in the strapi's studio itself right?

**Comment by gupta-ji6** [comment]
> Doesn't strapi itself have that feature where a document is initially saved as a draft and you publish it whenever you want. Both of you can review in the strapi's studio itself right?

yes, @hussamkhatib Strapi does have that feature and we use it too. But we don't store all resources on Strapi yet (only the once published or the next draft issue) and maintain them on Notion instead.
We collect potential resources on Notion database, review and pick them up for issues and once approved by both for an issue - we add it to Strapi draft.

We aren't storing everything on Strapi for cost-cutting you can say.

**Comment by gupta-ji6** [comment]
Also, syncig Notion DB with Strapi is a _good-to-have_ for us to reduce redundancy, not a necessity. Hence, we ourselves haven't yet explored much on this and made a little POC only.

**Comment by hussamkhatib** [comment]
cool, I will go through the strapi webhooks guide and let you know If I want to work on it.

**Comment by alexanderbnelson** [comment]
Would love to hear more about syncing a Notion db with Strapi. Possible to see the implementation?

**Comment by gupta-ji6** [comment]
> Would love to hear more about syncing a Notion db with Strapi. Possible to see the implementation?

hi @alexanderbnelson, we haven't prioritised working on it yet & hence there is no implementation I can share as of now. We only explored Notion API, webhooks & Strapi API's to have a flow overview if this could actually work.

We'll update this issue thread whenever we work on it.
